src/cityreader/cityreader.py

```diff
@@ -17,17 +21,45 @@
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the 
+    # `cities` list
+    import csv
+    # This website was helpful:
+    # https://www.geeksforgeeks.org/working-csv-files-python/
+    # Though it looks like it's out of date,
+    # since I need the double underscores around "next"
+    # (discovred by looking at the documentation here--
+    # https://docs.python.org/3/library/csv.html)
+    with open('cities.csv', 'r') as csvfile:
+        cityreader = csv.reader(csvfile)
+        fields = cityreader.__next__()
```

Comment:
Very nice solution! It is a little difficult to read through upon first glance.
Alternate solutions: 
I used to delimiter and a count to parse my file and skip the first line. 
Here is the solution from the instructors: 
```
    with open('cities.csv', newline='') as csvfile:
        reader = csv.reader(csvfile)

        # Advance the reader one row to skip the header
        next(reader)

        for row in reader:
            name = row[0]
            lat = float(row[3])
            lon = float(row[4])

            cities.append(City(name, lat, lon))
    return cities
```